messaging/src/test/java/org/axonframework/eventhandling/TimestampParameterResolverFactoryTest.java

```diff
@@ -16,32 +16,39 @@
 
 package org.axonframework.eventhandling;
 
-import org.axonframework.messaging.annotation.ParameterResolver;
-import org.junit.Before;
-import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
 
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 import java.lang.reflect.Method;
 import java.time.Instant;
 import java.time.temporal.Temporal;
 
-import static org.junit.Assert.*;
+import org.axonframework.messaging.annotation.ParameterResolver;
+import org.junit.Before;
+import org.junit.Test;
 
 /**
  * @author Allard Buijze
  */
 public class TimestampParameterResolverFactoryTest {
 
     private TimestampParameterResolverFactory testSubject;
-    private Timestamp annotation;
     private Method instantMethod;
     private Method temporalMethod;
     private Method stringMethod;
     private Method nonAnnotatedInstantMethod;
+	private Method metaAnnotatedMethod;
```

Comment:
Indentation seems a little off.
Would you mind making all your changes correctly indented with the rest of this class?